### PR TITLE
Fix multiple .schelp files (grammar, missing args, broken links, missing methods)

### PR DESCRIPTION
**Pull request by brianlheim**
I hope it's okay that I bundled these all together--most are pretty uncontroversial, and I figured it was better than making 20 separate PRs.

I've been using this code to find SCDoc warnings:
```
a = SCDoc.indexAllDocuments;
fork {a.documents.keys.do {|x| HelpBrowser.openHelpFor(x); 0.05.wait}};
```

The majority of these commits fix the issues that generate those warnings. The rest are grammar fixes and similar issues that I discovered during that process. More than happy to discuss and/or walk back changes.

### CODE REVIEW

File: HelpSource/Classes/RawArray.schelp [hunk 1]
```diff
@@ -8,6 +8,3 @@ RawArray is the abstract superclass of a group of array classes that hold raw da
 INSTANCEMETHODS::
 
 private::archiveAsCompileString, archiveAsObject
-
-method::write
-Writes the array as a file.
```
Comment (telephon) [inline HelpSource/Classes/RawArray.schelp]: "to a file" might be better?

Comment (brianlheim) [inline HelpSource/Classes/RawArray.schelp]: I'm deleting this method though, because it doesn't exist on RawArray. :P

Comment (telephon) [inline HelpSource/Classes/RawArray.schelp]: dah! :)

**Comment by snappizz** [comment]
I'll take a look soon.

**Comment by adcxyz** [comment]
thanks brian - only found one little glitch, fixed and merged.

**Comment by brianlheim** [comment]
@adcxyz thanks!

**Pull request merged**